### ISSUE TITLE
The docker version of arm64 won't work due to a mistake in Dockerfile

### ISSUE THREAD

**Issue by qqice233** [cdr/code-server#1481]
When I tried to run the arm64 version with docker, I always ran into an error saying 
`/usr/local/bin/fixuid: 2: /usr/local/bin/fixuid: Syntax error: word unexpected (expecting ")")`
Then I looked into the Image layers in Portainer, and found:
`RUN curl -SsL https://github.com/boxboat/fixuid/releases/download/v0.4/fixuid-0.4-linux-amd64.tar.gz | tar -C /usr/local/bin -xzf - && chown root:root /usr/local/bin/fixuid && chmod 4755 /usr/local/bin/fixuid && mkdir -p /etc/fixuid && printf "user: coder\ngroup: coder\n" > /etc/fixuid/config.yml`
Basically, the amd64 version of fixuid can't be run on arm64 devices, which caused the error.
So I have built the arm64 version of fixuid myself:
[https://github.com/qqice233/fixuid/releases/tag/v0.41](https://github.com/qqice233/fixuid/releases/tag/v0.41)
Please test it and fix the problem in Dockerfile.

Thanks.

**Comment by nhooyr** [comment]
Thanks for reporting, see #1505
